processOptions.go

```diff
@@ -248,6 +248,11 @@ func processOverrides() (tasks.Options, []override) {
 		options.Options["YesToAll"] = "true"
 	}
 
+	if config.Flags.K8sNamespace != "" {
```

Comment:
if not set the default it is an empty string right?

```
	defaultString := ""
	flag.StringVar(&Flags.K8sNamespace, "k8s-namespace", defaultString, "Specify a namespace to use when executing the kubectl command")

```

Reply:
Yes it is.

Reply:
So it gathers everyting from your actual context, maybe it's an unwanted behavior.

Reply:
🤔 maybe you are right, this is the behaviour that I would expect